client/components/theme-preview-modal/style.scss

```diff
@@ -0,0 +1,192 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/variables";
+
+$break-design-preview: 1024px;
```

Comment:
It would be better to use one of the breakpoints defined [here](https://github.com/WordPress/gutenberg/blob/4ccdee3b0004ca9aa464151b93dfdced8502dcac/packages/base-styles/_breakpoints.scss) and avoid declaring this custom breakpoint over and over.

I'll create a follow-up PR to consolidate this breakpoint into `break-xlarge` (1080px) cc: @SaxonF

Reply:
#71655 71655